src/components/books/BookDetails.vue

```diff
@@ -99,6 +105,9 @@ export default {
     hasLastUpdated() {
       return this.item.hasLastUpdated;
     },
+    hasInstitutions() {
+      return this.item.institutions && this.item.author.length > 0;
```

Comment:
Shouldn't this be checking against institutions?

```suggestion
      return this.item.institutions && this.item.institutions.length > 0;
```